packages/client/src/store/modules/auth/reducer.ts

```diff
@@ -0,0 +1,62 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit'
+import { AUTHORIZATION_STATUS } from '../../../utils/consts'
+import { AuthState, RootState } from '../../types'
+import yApiService from '../../../services/y-api-service'
+import { message } from 'antd'
+
+const initialState: AuthState = {
+  id: null,
+  display_name: null,
+  email: null,
+  login: null,
+  phone: null,
+  first_name: null,
+  second_name: null,
+  avatar: null,
+  password: null,
+  authorizedStatus: AUTHORIZATION_STATUS.UNKNOWN,
+}
+
+export const fetchUserData = createAsyncThunk(
+  'auth/fetchUserData',
+  async () => {
+    try {
+      const response = await yApiService.getUser()
+
+      return response
+    } catch (error) {
+      message.error(`Ошибка загрузки файла: ${error}`)
+    }
+  }
+)
```

Comment:
```suggestion
export const fetchUserData = createAsyncThunk(
  'auth/fetchUserData',
  async (_, { rejectWithValue }) => {
    try {
      const response = await yApiService.getUser()

      return response
    } catch (error) {
      message.error(`Ошибка загрузки файла: ${error}`)
      return rejectWithValue(err.response.data)
    }
  }
)
```

иначе в экстраРедьюсер с ошибкой не попадет